composer.json

```diff
@@ -17,7 +17,7 @@
         "hubspot"
     ],
     "require": {
-        "league/oauth2-client": "^1.0"
+        "league/oauth2-client": "^2.0"
```

Comment:
What version is this release targeting?  Looks like it's at `0.1.1`.  I think we should perhaps target v1.0 since a dependency is now requiring a major bump.

Reply:
Yep, I was going to bump to v1.0 when releasing with this change :+1: